### PR TITLE
Check the users balance before trying to publish/create a channel

### PR DESCRIPTION
**Pull request by seanyesmunt**
Also added a start script just for fun.

Mostly a test to see what it would look like. Not sure if we want to go this route, or go through the daemon for everything

### CODE REVIEW

File: src/renderer/js/component/publishForm/index.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
 import React from "react";
 import { connect } from "react-redux";
 import PublishForm from "./view";
+import { selectBalance } from "redux/selectors/wallet";
+import { formatFullPrice } from "util/formatCredits";
 
-export default connect(null, null)(PublishForm);
+const select = state => ({
+  balance: formatFullPrice(selectBalance(state) || 0, 10),
```
Comment (kauffj) [inline src/renderer/js/component/publishForm/index.js]: Why format this? Also we should probably trust selector to fallback to 0?

**Comment by kauffj** [comment]
@seanyesmunt does the start script just make it so you can run the app with `npm start`?

File: src/renderer/js/component/publishForm/index.js (hunk 1 above)
Comment (seanyesmunt) [reply src/renderer/js/component/publishForm/index.js]: Ignore this for now. Just copy/pasted from somewhere else

**Comment by seanyesmunt** [comment]
Yeah. Or "yarn start". Still need to build before running it

File: package.json [hunk 2]
```diff
@@ -66,6 +66,9 @@
       "perMachine": true
     }
   },
+  "scripts": {
+    "start": "./node_modules/.bin/electron src/main"
```
Comment (liamcardenas) [inline package.json]: does it work to just say "electron src/main" since electron is in the bin?

Comment (seanyesmunt) [reply package.json]: It does, I like to show the full path so others know exactly whats going on

Comment (seanyesmunt) [reply package.json]: I can change it to `electron src/main` if you think that looks better

Comment (liamcardenas) [reply package.json]: hmm no need, lets keep it as is then

**Pull request merged**